subscribers:

  queue_size: 5

  object_recognition:
    topic: /butia_vision/or/object_recognition

  face_recognition:
    topic: /butia_vision/fr/face_recognition

  people_tracking:
    topic: /butia_vision/pt/people_tracking

publishers:

  queue_size: 5

  object_recognition:
    topic: /butia_vision/or/object_recognition3d

  face_recognition:
    topic: /butia_vision/fr/face_recognition3d

  people_tracking:
    topic: /butia_vision/pt/people_tracking3d

clients:

  image_request:
    service: /butia_vision/bvb/image_request
